cli/dcoscli/experimental/main.py

```diff
@@ -334,3 +337,12 @@ def _is_local_reference(item):
     :rtype: bool
     """
     return isinstance(item, six.string_types) and item.startswith("@")
+
+
+def _service_start(package_name, package_version, options_path):
+    manager = servicemanager.ServiceManager()
+    options = None if not options_path else get_user_options(options_path)
```

Comment:
Actually, it looks like `get_user_options()` handles the case where the path is `None`.